megamek/src/megamek/common/verifier/TestEntity.java

```diff
@@ -1354,6 +1354,19 @@ public boolean hasIllegalEquipmentCombinations(StringBuffer buff) {
                     || m.getType().hasFlag(MiscType.F_CASPAR) || m.getType().hasFlag(MiscType.F_CASPARII)) {
                 robotics++;
             }
+            if (m.getType().hasFlag(MiscType.F_LIFTHOIST)) {
+                liftHoists++;
+            } else if ((m.getLocation() > 0)
+                    && ((m.getType().hasFlag(MiscType.F_CLUB) && !((MiscType) m.getType()).isShield())
+                    || m.getType().hasFlag(MiscType.F_BULLDOZER)
+                    || m.getType().hasFlag(MiscType.F_HAND_WEAPON))) {
+                physicalWeaponsByLocation.putIfAbsent(m.getLocation(), new ArrayList<>());
+                physicalWeaponsByLocation.get(m.getLocation()).add(m.getType());
```

Comment:
Any objection to?
```suggestion
                physicalWeaponsByLocation.computeIfAbsent(m.getLocation(), new ArrayList<>()).add(m.getType());
```

Reply:
The second parameter of computeIfAbsent is a Function, so it needs to be
```
physicalWeaponsByLocation.computeIfAbsent(m.getLocation(), ArrayList::new).add(m.getType());

Reply:
Apologies, attempted the suggestion from memory.